submissions/DmytroMukhin/HTML_CSS_Hooli-style_Popup/index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css" />
+    <title>HTML CSS практика: Hooli-style Popup</title>
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+          <nav class="navigation">
+            <ul class="navigation__list">
+              <li class="navigation__item">
+                <a class="navigation__link" href="#">mail</a>
+              </li>
+              <li class="navigation__item">
+                <a class="navigation__link" href="#">images</a>
+              </li>
+              <li class="navigation__item navigation__item--popup">
+                <div class="popup">  
+                  <input
+                    class="popup__input"
+                    name="input-popup"
+                    id="popup__input--id"
+                    type="checkbox"
+                   />
+                  <label class="popup__label" for="popup__input--id"></label>
+                  <ul class="popup__list">
+                    <li class="popup__item">
+                      <a class="popup__link" href="#">
+                        <figure class="popup__figure">
+                          <img
+                            class="popup__img"
+                            src="icons/calendar.png"
+                            alt=""
+                            width="40"
```

Comment:
why do you need inline style here?

Reply:
I tested with the size and forgot to remove.